default.yml

```diff
@@ -1,17 +1,15 @@
-# For Fleet Free, this file updates policies, queries, agent_options, and controls for all hosts.
+# For Fleet Free, this file updates policies, queries, and agent_options for all hosts.
 
-# For Fleet Premium, the default.yml file updates policies and queries that run on all hosts ("All teams"). Controls and agent options are updated for hosts on "No team."
+# For Fleet Premium, the default.yml file updates policies and queries that run on all hosts ("All teams"). 
 
 policies:
 queries:
 agent_options:
   path: ./lib/agent-options.yml
-controls:
```

Comment:
If we remove controls, how will the Fleet Free user add configuration profiles?

Maybe we instruction Fleet Premium users only to remove `controls`?

Reply:
Good point, I forgot about Fleet Free, we can leave it here empty.